app/models/project.rb

```diff
@@ -448,8 +455,8 @@ def out_of_range_observations
     elsif end_date.nil?
       observations.where(Observation[:when] < start_date)
     else
-      observations.where(Observation[:when] > end_date).
-        or(observations.where(Observation[:when] < start_date))
+      observations.reorder("").where(Observation[:when] > end_date).
```

Comment:
`reorder("")` strikes me as a weird thing to do here and an unneeded complication.